app/page.js

```diff
@@ -9,24 +9,83 @@ export default function Home() {
   const { playerType, setPlayerType, gameType, setGameType } =
     useContext(GameContext);
 
-  const className = `text-xl ${playerType === "X" ? "bg-yellow" : "bg-blue"}`;
-
   return (
-    <>
-      <div className="flex gap-4">
-        <button className={className} onClick={() => setPlayerType("X")}>
-          X
-        </button>
-        <button className={className} onClick={() => setPlayerType("O")}>
-          O
-        </button>
+    <div className="flex flex-col items-center p-6 w-full justify-center max-w-[28.75rem]">
+      <svg width="72" height="32" xmlns="http://www.w3.org/2000/svg">
+        <g fill="none" fillRule="evenodd">
+          <path
+            d="M8.562 1.634 16 9.073l7.438-7.439a3 3 0 0 1 4.243 0l2.685 2.685a3 3 0 0 1 0 4.243L22.927 16l7.439 7.438a3 3 0 0 1 0 4.243l-2.685 2.685a3 3 0 0 1-4.243 0L16 22.927l-7.438 7.439a3 3 0 0 1-4.243 0L1.634 27.68a3 3 0 0 1 0-4.243L9.073 16 1.634 8.562a3 3 0 0 1 0-4.243L4.32 1.634a3 3 0 0 1 4.243 0Z"
+            fill="#31C3BD"
+          />
+          <path
+            d="M56.1 0c8.765 0 15.87 7.106 15.87 15.87 0 8.766-7.105 15.871-15.87 15.871-8.765 0-15.87-7.105-15.87-15.87C40.23 7.106 47.334 0 56.1 0Zm0 9.405a6.466 6.466 0 1 0 0 12.931 6.466 6.466 0 0 0 0-12.931Z"
+            fill="#F2B137"
+            fillRule="nonzero"
+          />
+        </g>
+      </svg>
+      <div className="flex flex-col items-center justify-center gap-4 bg-semiDarkNavy rounded-lg p-6 w-full my-8 shadow-[0px_-8px_0px_0px_#10212A_inset]">
+        <span className="text-base font-bold">PICK PLAYER 1’S MARK</span>
+        <div className="flex items-center justify-center gap-4 bg-darkNavy rounded-lg w-full p-2 ">
+          <button
```

Comment:
These two button can be merged into one. You should check with `ternary`: 
- classes
- onClick functions
- SVGs.